src/js/smooch.jsx

```diff
@@ -161,6 +164,14 @@ export class Smooch {
                 }
             });
         }).then((loginResponse) => {
+            observable.on('socket:connected', () => {
```

Comment:
I don't know if I agree that is should resolve when this is triggered, sometimes it'll get triggered and it's unrelated to login